<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Line Chart </string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="11" row="0" column="2" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="chartFrame" >
      <property name="minimumSize" >
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QCheckBox" name="paintValuesCB" >
     <property name="text" >
      <string>Display Data Values</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lineTypeLB" >
       <property name="text" >
        <string>Line Chart Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lineTypeCB" >
       <item>
        <property name="text" >
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Stacked</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Percent</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="threeDLabel" >
         <property name="text" >
          <string>ThreeD Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="threeDModeCB" >
         <property name="text" >
          <string>ThreeD Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="depthLabel" >
         <property name="text" >
          <string>ThreeD Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="depthSB" >
         <property name="maximum" >
          <number>99</number>
         </property>
         <property name="value" >
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="Line" name="line_4" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="penLabel" >
       <property name="text" >
        <string>Show Areas</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="highlightAreaCB" >
         <property name="text" >
          <string>Highlight Area</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="highlightAreaSB" >
         <property name="maximum" >
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="0" >
    <widget class="QPushButton" name="pushButtonSave" >
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QPushButton" name="pushButtonLoad" >
     <property name="text" >
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>201</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="LineChart" />
 </resources>
 <connections/>
</ui>
